dist2src/core.py

```diff
@@ -662,7 +670,15 @@ def add_packit_config(
             "sources": self.get_lookaside_sources(lookaside_branch),
         }
 
-        self.source_git_path.joinpath(".packit.yaml").write_text(dump(config))
+        self.source_git_path.joinpath(".packit.yaml").write_text(
+            yaml.dump_all(
+                [config],
+                Dumper=SafeDumperWithoutAliases,
+                # True in el8, False in latest pyyam
+                # dumps things into a block instead of ugly inline
+                default_flow_style=False,
```

Comment:
Is the comment trying to say that `default_flow_style=True` (default in `el8`) creates ugly inline while `default_flow_style=False` (default in latest pyyml) creates nice block?

Reply:
very good question :)

Reply:
I can now confirm this is indeed the case: I'll make the message more verbose and hopefully understable